index.bs

```diff
@@ -1164,8 +1166,15 @@ The dictionary SHALL have the following attributes:
 : <dfn for="WebTransportStats" dict-member>packetsReceived</dfn>
 :: The number of total packets received on the QUIC connection, including
    packets that were not processable.
+: <dfn for="WebTransportStats" dict-member>smoothedRtt</dfn>
+:: The smoothed round-trip time currently observed on the connection, as defined
```

Comment:
```suggestion
:: The smoothed round-trip time (RTT) currently observed on the connection, as defined
```

Having the acronym spelled out is probably useful given that we use it in variable names.